go/pkg/rexec/rexec.go

```diff
@@ -120,11 +120,19 @@ func (ec *Context) setOutputMetadata() {
 		ec.Metadata.TotalOutputBytes += int64(len(ec.resPb.StdoutRaw))
 	} else if ec.resPb.StdoutDigest != nil {
```

Comment:
The digest is always set whereas the raw content may not always be set. In other words, this branch will be skipped if both are set. However, we always want to capture the digest.